We do not want framework lookups, because that influences the searchpath
and things like sys.prefix.  On a non-framework install this differs
almost nothing, but avoids weird behaviour if a framework suddenly
somehow becomes available.  On a framework install this makes python
behave like a normal UNIX install.

--- Python-2.7/Modules/getpath.c
+++ Python-2.7/Modules/getpath.c
@@ -446,7 +446,7 @@
         strncpy(argv0_path, progpath, MAXPATHLEN);
         argv0_path[MAXPATHLEN] = '\0';
 
-#ifdef WITH_NEXT_FRAMEWORK
+#if 0
         /* On Mac OS X we have a special case if we're running from a framework.
         ** This is because the python home should be set relative to the library,
         ** which is in the framework, not relative to the executable, which may
